### ISSUE TITLE
possible bug in resDS for some methods

### ISSUE THREAD

**Issue by AnjaliC4** [HelenaLC/muscat#102]
Hi, In resDS function - you calculate cpm from y$data[[k]] .This will work for edgeR as this returns normalized count but will not work when using limma-voom. -
as.matrix(y$data[[k]]) returns "E" matrix of limma-voom which are already logCMP values I believe? Please confirm.

Thanks

**Comment by AnjaliC4** [comment]
Hi, can you please confirm the above or let me know if my understanding is correct. Thank you.

**Comment by HelenaLC** [comment]
You're right! Thanks a lot for catching this. I am pushing a fix to rectify; i.e., calling `cpm()` only when `method != "limma-voom"`, where `limma-voom` will do `2^E`, which are on the log2-scale per `voom` documentation, as you suggested.
